pkg/operator/ceph/cluster/osd/topology/topology.go

```diff
@@ -39,6 +40,8 @@ var (
 
 	// The list of supported failure domains in the CRUSH map, ordered from lowest to highest
 	CRUSHMapLevelsOrdered = append([]string{"host"}, append(CRUSHTopologyLabels, KubernetesTopologyLabels...)...)
+
+	logger = capnslog.NewPackageLogger("github.com/rook/rook", "op-osd")
```

Comment:
Not sure if `op-osd` is the correct value here. Currently, this package is only referred to by the `osd` package. Let me know if this should be changed.

Reply:
It's not critical, but how about this? It's nice if each package can have a unique value to simplify log analysis
```suggestion
	logger = capnslog.NewPackageLogger("github.com/rook/rook", "osd-topology")
```